elyra/metadata/schemas/runtime-image.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Schema for Runtime Image metadata",
+  "name": "runtime-image",
+  "properties": {
```

Comment:
This schema will produce a test failure because it doesn't have a `namespace` property.  This property should be:
```json
{
"namespace": "runtime-images"
}
```
#408 adds a test (`test_validate_factory_schema`) which checks our schema files for the minimum items we expect at a system level.

Reply:
Could you elaborate more on what is needed here? I looked at the schemas on #408 and didn't see the namespace field? Maybe I overlooked something?

Reply:
Sorry - I meant #480.
https://github.com/elyra-ai/elyra/blob/5f305a6c5a31593c6b6a7d2e746500eafa84611f/elyra/metadata/schemas/kfp.json#L5